### PR TITLE
Simple botany logging

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Adds in a simple botany log file that logs the following things:
 - Slips
 - - Bluespace teleports
 - Squashing
 - - Smoke Generation
 - - Seperated Chemicals priming
 - Plant mutations
 - Plants being dropped
 - Plants harvested
 - Plants dug up

All plant stats will also be shown when interactions with plants are had

## Why It's Good For The Game

Botany grief is painfully easy and painfully hard to find who did what.

```
[2020-07-30 16:45:17.392] Starting up round ID .
 - -------------------------
[2020-07-30 16:47:43.199] Unstable mutagen applied to the hydroponics tray (Ambrosia Vulgaris) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 134, 2) using unstable mutagen.
[2020-07-30 16:47:43.759] Unstable mutagen applied to the hydroponics tray (Ambrosia Vulgaris) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 134, 2) using unstable mutagen.
[2020-07-30 16:47:44.241] The pack of ambrosia vulgaris seeds changed species into the pack of ambrosia deus seeds by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 134, 2) using unstable mutagen.
 - =====Old Seed=====: 
 - ==Plant Stats==
 - Name: Ambrosia Vulgaris
 - Type: /obj/item/seeds/ambrosia
 - ==GENES==
 - GENE <Lifespan 64>
 - GENE <Endurance 34>
 - GENE <Weed Growth Rate 0>
 - GENE <Weed Vulnerability 8>
 - GENE <Yield 9>
 - GENE <Production Speed 7>
 - GENE <Potency 0>
 - GENE <Perennial Growth>
 - GENE <Space drugs production 15%>
 - GENE <Bicaridine production 10%>
 - GENE <Kelotane production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 5%>
 - GENE <Toxin production 10%>)}
 - =====New seed:===== 
 - ==Plant Stats==
 - Name: Ambrosia Deus
 - Type: /obj/item/seeds/ambrosia/deus
 - ==GENES==
 - GENE <Lifespan 60>
 - GENE <Endurance 23>
 - GENE <Weed Growth Rate 0>
 - GENE <Weed Vulnerability 6>
 - GENE <Yield 9>
 - GENE <Production Speed 8>
 - GENE <Potency 0>
 - GENE <Perennial Growth>
 - GENE <Omnizine production 15%>
 - GENE <Synaptizine production 15%>
 - GENE <Space drugs production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 5%>)}
 - 
[2020-07-30 16:47:44.263] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 134, 2) using unstable mutagen.
[2020-07-30 16:47:45.278] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 134, 2) using unstable mutagen.
[2020-07-30 16:47:45.709] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 134, 2) using unstable mutagen.
[2020-07-30 16:47:46.616] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 134, 2) using unstable mutagen.
[2020-07-30 16:47:54.889] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:47:55.560] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:47:56.194] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:47:56.716] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:47:57.489] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:47:58.058] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:47:58.594] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:48:08.392] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:48:08.593] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:48:09.278] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:48:09.725] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:48:10.049] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:48:10.293] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:48:10.561] Unstable mutagen applied to the hydroponics tray (Ambrosia Deus) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:48:10.790] The pack of ambrosia deus seeds changed species into the pack of ambrosia gaia seeds by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
 - =====Old Seed=====: 
 - ==Plant Stats==
 - Name: Ambrosia Deus
 - Type: /obj/item/seeds/ambrosia/deus
 - ==GENES==
 - GENE <Lifespan 59>
 - GENE <Endurance 14>
 - GENE <Weed Growth Rate 8>
 - GENE <Weed Vulnerability 24>
 - GENE <Yield 10>
 - GENE <Production Speed 10>
 - GENE <Potency 0>
 - GENE <Perennial Growth>
 - GENE <Omnizine production 15%>
 - GENE <Synaptizine production 15%>
 - GENE <Space drugs production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 5%>)}
 - =====New seed:===== 
 - ==Plant Stats==
 - Name: Ambrosia Gaia
 - Type: /obj/item/seeds/ambrosia/gaia
 - ==GENES==
 - GENE <Lifespan 64>
 - GENE <Endurance 21>
 - GENE <Weed Growth Rate 6>
 - GENE <Weed Vulnerability 64>
 - GENE <Yield 2>
 - GENE <Production Speed 4>
 - GENE <Potency 0>
 - GENE <Earthsblood production 5%>
 - GENE <Nutriment production 6%>
 - GENE <Vitamin production 5%>)}
 - 
[2020-07-30 16:48:10.843] Unstable mutagen applied to the hydroponics tray (Ambrosia Gaia) by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 135, 2) using unstable mutagen.
[2020-07-30 16:50:32.368] *no key*/(Kaylin Waldron) dropped the banana at Starboard Bow Maintenance (0, 0, 0)
 - ==Plant Stats==
 - Name: Banana Tree
 - 
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 50>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:51:29.855] The pack of ambrosia gaia seeds harvested by PowerfulBacon/(Michaela Powers) at (Hydroponics (163, 134, 2))
 - ==Plant Stats==
 - Name: Ambrosia Gaia
 - Type: /obj/item/seeds/ambrosia/gaia
 - ==GENES==
 - GENE <Lifespan 64>
 - GENE <Endurance 19>
 - GENE <Weed Growth Rate 2>
 - GENE <Weed Vulnerability 59>
 - GENE <Yield 1>
 - GENE <Production Speed 4>
 - GENE <Potency 0>
 - GENE <Earthsblood production 5%>
 - GENE <Nutriment production 6%>
 - GENE <Vitamin production 5%>)}
 - 
[2020-07-30 16:51:29.856] The pack of ambrosia gaia seeds harvested by PowerfulBacon/(Michaela Powers) at Hydroponics (162, 134, 2)
 - ==Plant Stats==
 - Name: Ambrosia Gaia
 - Type: /obj/item/seeds/ambrosia/gaia
 - ==GENES==
 - GENE <Lifespan 64>
 - GENE <Endurance 19>
 - GENE <Weed Growth Rate 2>
 - GENE <Weed Vulnerability 59>
 - GENE <Yield 1>
 - GENE <Production Speed 4>
 - GENE <Potency 0>
 - GENE <Earthsblood production 5%>
 - GENE <Nutriment production 6%>
 - GENE <Vitamin production 5%>)}
 - 
[2020-07-30 16:51:31.688] PowerfulBacon/(Michaela Powers) dropped the ambrosia gaia branch at Hydroponics (0, 0, 0)
 - ==Plant Stats==
 - Name: Ambrosia Gaia
 - powerfulbacon
 - Type: /obj/item/seeds/ambrosia/gaia
 - ==GENES==
 - GENE <Lifespan 64>
 - GENE <Endurance 19>
 - GENE <Weed Growth Rate 2>
 - GENE <Weed Vulnerability 59>
 - GENE <Yield 1>
 - GENE <Production Speed 4>
 - GENE <Potency 0>
 - GENE <Earthsblood production 5%>
 - GENE <Nutriment production 6%>
 - GENE <Vitamin production 5%>)}
 - 
[2020-07-30 16:51:35.693] PowerfulBacon/(Michaela Powers) dropped the ambrosia gaia branch at Hydroponics (0, 0, 0)
 - ==Plant Stats==
 - Name: Ambrosia Gaia
 - powerfulbacon
 - Type: /obj/item/seeds/ambrosia/gaia
 - ==GENES==
 - GENE <Lifespan 64>
 - GENE <Endurance 19>
 - GENE <Weed Growth Rate 2>
 - GENE <Weed Vulnerability 59>
 - GENE <Yield 1>
 - GENE <Production Speed 4>
 - GENE <Potency 0>
 - GENE <Earthsblood production 5%>
 - GENE <Nutriment production 6%>
 - GENE <Vitamin production 5%>)}
 - 
[2020-07-30 16:51:35.694] PowerfulBacon/(Michaela Powers) dropped the ambrosia gaia branch at Hydroponics (0, 0, 0)
 - ==Plant Stats==
 - Name: Ambrosia Gaia
 - powerfulbacon
 - Type: /obj/item/seeds/ambrosia/gaia
 - ==GENES==
 - GENE <Lifespan 64>
 - GENE <Endurance 19>
 - GENE <Weed Growth Rate 2>
 - GENE <Weed Vulnerability 59>
 - GENE <Yield 1>
 - GENE <Production Speed 4>
 - GENE <Potency 0>
 - GENE <Earthsblood production 5%>
 - GENE <Nutriment production 6%>
 - GENE <Vitamin production 5%>)}
 - 
[2020-07-30 16:51:39.445] The pack of watermelon seeds harvested by PowerfulBacon/(Michaela Powers) at (Hydroponics (156, 134, 2))
 - ==Plant Stats==
 - Name: Watermelon Vines
 - Type: /obj/item/seeds/watermelon
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 40>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Perennial Growth>
 - GENE <Water production 20%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 20%>)}
 - 
[2020-07-30 16:51:39.445] The pack of watermelon seeds harvested by PowerfulBacon/(Michaela Powers) at Hydroponics (157, 134, 2)
 - ==Plant Stats==
 - Name: Watermelon Vines
 - Type: /obj/item/seeds/watermelon
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 40>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Perennial Growth>
 - GENE <Water production 20%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 20%>)}
 - 
[2020-07-30 16:51:39.870] The pack of banana seeds harvested by PowerfulBacon/(Michaela Powers) at (Hydroponics (156, 135, 2))
 - ==Plant Stats==
 - Name: Banana Tree
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:51:39.871] The pack of banana seeds harvested by PowerfulBacon/(Michaela Powers) at Hydroponics (157, 135, 2)
 - ==Plant Stats==
 - Name: Banana Tree
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:51:43.320] PowerfulBacon/(Michaela Powers) dropped the banana at Hydroponics (0, 0, 0)
 - ==Plant Stats==
 - Name: Banana Tree
 - powerfulbacon
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:51:43.321] PowerfulBacon/(Michaela Powers) dropped the banana at Hydroponics (0, 0, 0)
 - ==Plant Stats==
 - Name: Banana Tree
 - powerfulbacon
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:51:43.918] PowerfulBacon/(Michaela Powers) dropped the watermelon at Hydroponics (0, 0, 0)
 - ==Plant Stats==
 - Name: Watermelon Vines
 - powerfulbacon
 - Type: /obj/item/seeds/watermelon
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 40>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Perennial Growth>
 - GENE <Water production 20%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 20%>)}
 - 
[2020-07-30 16:51:43.919] PowerfulBacon/(Michaela Powers) dropped the watermelon at Hydroponics (0, 0, 0)
 - ==Plant Stats==
 - Name: Watermelon Vines
 - powerfulbacon
 - Type: /obj/item/seeds/watermelon
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 40>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Perennial Growth>
 - GENE <Water production 20%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 20%>)}
 - 
[2020-07-30 16:52:12.022] PowerfulBacon/(Michaela Powers) dropped the watermelon at Hydroponics (158, 134, 2)
 - ==Plant Stats==
 - Name: Watermelon Vines
 - powerfulbacon
 - Type: /obj/item/seeds/watermelon
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 40>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Perennial Growth>
 - GENE <Water production 20%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 20%>)}
 - 
[2020-07-30 16:52:12.656] PowerfulBacon/(Michaela Powers) dropped the watermelon at Hydroponics (158, 135, 2)
 - ==Plant Stats==
 - Name: Watermelon Vines
 - powerfulbacon
 - Type: /obj/item/seeds/watermelon
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 40>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Perennial Growth>
 - GENE <Water production 20%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 20%>)}
 - 
[2020-07-30 16:52:15.824] PowerfulBacon/(Michaela Powers) dropped the banana at Hydroponics (0, 0, 0)
 - ==Plant Stats==
 - Name: Banana Tree
 - powerfulbacon
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:52:16.936] PowerfulBacon/(Michaela Powers) dropped the banana peel at Hydroponics (161, 135, 2)
 - ==Plant Stats==
 - Name: Banana Tree
 - 
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:52:17.824] Michaela Powers slipped on the banana peel at Hydroponics (161, 135, 2)
 - ==Plant Stats==
 - Name: Banana Tree
 - 
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:52:58.538] Michaela Powers slipped on the banana peel at Hydroponics (161, 135, 2)
 - ==Plant Stats==
 - Name: Banana Tree
 - 
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
[2020-07-30 16:53:03.135] PowerfulBacon/(Michaela Powers) dropped the banana peel at Hydroponics (161, 135, 2)
 - ==Plant Stats==
 - Name: Banana Tree
 - powerfulbacon
 - Type: /obj/item/seeds/banana
 - ==GENES==
 - GENE <Lifespan 50>
 - GENE <Endurance 30>
 - GENE <Weed Growth Rate 1>
 - GENE <Weed Vulnerability 5>
 - GENE <Yield 3>
 - GENE <Production Speed 6>
 - GENE <Potency 10>
 - GENE <Slippery Skin>
 - GENE <Perennial Growth>
 - GENE <Banana Juice production 10%>
 - GENE <Potassium production 10%>
 - GENE <Vitamin production 4%>
 - GENE <Nutriment production 2%>)}
 - 
```

## Changelog
:cl:
add: Botany.log
/:cl:

### CODE REVIEW

**Comment by yorii** [comment]
Smoke Generation <- this is already logged from the smoke proc btw
It would probably generate duplicate logs now, which I guess isn't really a big issue.

**Comment by PowerfulBacon** [comment]
> Smoke Generation <- this is already logged from the smoke proc btw
> It would probably generate duplicate logs now, which I guess isn't really a big issue.

I know, but they are logged to different .log files

**Comment by yorii** [comment]
Oh sweet, you're putting botany in separate log file? That would be very helpful 👍

**Comment by qwertyquerty** [comment]
Could maybe use some reformating tbh but otherwise good

File: code/__HELPERS/botany_log.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//Helpers for botany logging
+/proc/get_plant_stats(obj/item/seeds/plant, obj/item/container)
+	. = "\n==Plant Stats==\nName: [plant.plantname]\n[container?"Last Fingerprint: [container.fingerprintslast]\n":""]Type: [plant.type]\n==GENES=="
+	for(var/datum/plant_gene/gene in plant.genes)
+		. += "\nGENE <[gene.get_name()]>"
```
Comment (qwertyquerty) [inline code/__HELPERS/botany_log.dm:5]: I'd reformat this somehow

**Review by qwertyquerty — APPROVED** [review]

File: code/__HELPERS/botany_log.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/__HELPERS/botany_log.dm:5]: How? Put the \n after the GENE?

**Comment by utilbird** [comment]
mfw no more slippery prickle polonium melons

File: code/__HELPERS/botany_log.dm (hunk 1 above)
Comment (yorii) [reply code/__HELPERS/botany_log.dm:5]: Show pic how it would look in notepad 😄

Comment (PowerfulBacon) [reply code/__HELPERS/botany_log.dm:5]: There is a log file generated in the pr desc

Comment (yorii) [reply code/__HELPERS/botany_log.dm:5]: I like it! But oh boy that's gonna be a big log file :D

**Pull request closed without merging**